docs/userGuide/usingPlugins.md

```diff
@@ -0,0 +1,99 @@
+<frontmatter>
+  footer: userGuideFooter.md
+  siteNav: userGuideSections.md
+</frontmatter>
+
+<include src="../common/header.md" />
+
+<div class="website-content">
+
+# Using Plugins
+
+A plugin is a user defined extension that can add custom features to MarkBind. Markbind plugins are `js` scripts that are loaded and run during the page generation.
+
+### Adding Plugins
+
+Plugins are stored in the `_markbind/plugins` folder which is generated on `init`. To use a plugin, place the `js` file in the `_markbind/plugins` folder and add the following options to the site configuration:
+
+- `plugins`: An array of plugin names to use
+- `context`: An object mapping plugin name to data provided to the plugin. Each plugin can have its own custom data.

```

Comment:
Hm list might be a little confusing since it's actually an object, how about `An object containing lists of parameters passed to each individual plugin.`?

Reply:
From the point of view of a non-technical user, it does not look like an "object" to them.

How about `a mapping of a list of parameters passed to each individual plugin`?

Also the full stop usage is not consistent .